pmd-core/src/main/java/net/sourceforge/pmd/MultiValuePropertyDescriptor.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd;
+
+import java.util.List;
+
+/**
+ * Specializes property descriptors for single valued descriptors. For this type of property, the return value of the
+ * {@link #type()} method must be the class literal of the type parameter of the interface {@link PropertyDescriptor}.
+ *
+ * @param <V> The type of value this descriptor works with. This is the type of the list's component.
```

Comment:
I wanted to be consistent with AbstractMultiValuePropertyDescriptor, because V is not the T of the PropertyDescriptor. But if you think it's best I'll change it